### PR TITLE
Rename osu to peppy

### PR DESCRIPTION
**Pull request by Nep-Timeline**

### CODE REVIEW

**Review by rootmelo92118 — APPROVED** [review]

**Review by rootmelo92118 — CHANGES_REQUESTED** [review]
Why?

**Comment by Nep-Timeline** [comment]
> Why?

https://blog.ppy.sh/
This link belongs to the blog and does not belong to the game 'osu'

**Comment by rootmelo92118** [comment]
So... is peppy a company?

File: data/osu
Comment (rootmelo92118) [inline data/osu:1]: If it does not have any subdomain name, you should use full prefix.

Comment (Nep-Timeline) [reply data/osu:1]: done

**Review by rootmelo92118 — APPROVED** [review]

**Comment by rootmelo92118** [comment]
Now you just need the last step.
Find a relevant category to include `peppy`.

File: data/category-orgs [hunk 1]
```diff
@@ -8,6 +8,7 @@ include:f-droid
 include:ieee
 include:nist
 include:openstreetmap
+include:peppy
```
Comment (rootmelo92118) [inline data/category-orgs:11]: sort it.

**Review by rootmelo92118 — CHANGES_REQUESTED** [review]

File: data/category-orgs (hunk 1 above)
Comment (Nep-Timeline) [reply data/category-orgs]: ok

**Review by rootmelo92118 — APPROVED** [review]

**Pull request merged**